graphdatascience/graph/graph_entity_ops_runner.py

```diff
@@ -1,7 +1,9 @@
 from typing import Any, Dict, List, Type, Union
 
+import pandas as pd
 from pandas import DataFrame, Series
```

Comment:
I think we should do either way of these imports. 
Leaning towards the `from` version :)